go/stream/message.go

```diff
@@ -144,6 +187,12 @@ func ToMapProperties(properties MessageProperties) map[string]string {
 		m[mapKeyWebhookFailureReason] = properties.WebhookFailureReason
 		m[mapKeyStage] = properties.Stage
 	}
+	if properties.IsBot {
+		m[mapKeyIsBot] = strconv.FormatBool(properties.IsBot)
```

Comment:
Given that you add it only if `IsBot` is `true` you could skip the formatting altogether.

```go
m[mapKeyIsBot] = "true"
```